tests/test_dimension_service.py

```diff
@@ -17,6 +18,15 @@ def test_create_dimension_on_measure_page(stub_measure_page):
     db_dimension = Dimension.query.all()[0]
     assert stub_measure_page.dimensions[0].guid == db_dimension.guid
     assert stub_measure_page.dimensions[0].title == db_dimension.title
+    assert not stub_measure_page.dimensions[0].created_at == None, "Should have set a creation timestamp"
```

Comment:
It's more pythonic to write this as `is not None` (same as with `True`/`False` tests). If we can update this it'd be good, otherwise we'll have to clean it up later when we stick a code linter in.